flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -1311,6 +1311,19 @@ public boolean updateState(TaskExecutionState state) {
 		}
 	}
 
+	private Map<String, Accumulator<?, ?>> deserializeAccumulators(TaskExecutionState state) {
```

Comment:
I've checked that returning `null` is the same behaviour as in `accumulators.deserializeUserAccumulators(userClassLoader)`. In general it would be good to add annotations for `@Nullable` return types or simply return empty collections, but that's an orthogonal change.